src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts

```diff
@@ -0,0 +1,124 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Sku } from '@azure/arm-postgresql/src/models';
+import { AzureWizardPromptStep, IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { ext } from '../../../../extensionVariables';
+import { localize } from '../../../../utils/localize';
+import { nonNullProp } from '../../../../utils/nonNull';
+import { IPostgresServerWizardContext } from '../IPostgresServerWizardContext';
+
+class SkuOption {
```

Comment:
nit: Our team usually use interfaces for this sort of thing
```suggestion
interface ISkuOption {
```